### DISCUSSION TITLE
VMWare - Hostname on Bottlerocket worker nodes

### DISCUSSION THREAD

**Discussion by trebonian** [bottlerocket-os/bottlerocket#1636] [Q&A] [answered]
I followed the Quickstart instructions for adding a Bottlerocket work to a cluster. It worked almost perfectly except that I can't seem to find a way to set the node name of the Bottlerocket node (which I believe is coming from the hostname). It seems to use the IP address of the node as the hostname. Is there any way to change this?

**Comment by zmrow** [comment, marked as answer]
Hi @trebonian !  We're currently working on making hostname something that can be set via user data in #1608 .  :)

**Reply by zmrow** [reply]
This was implemented in #1664 !
